INFO 2025-08-24 13:46:40,063 autoreload 6105 139781868523776 Watching for file changes with StatReloader
INFO 2025-08-24 13:46:40,276 autoreload 6107 140260849012992 Watching for file changes with StatReloader
INFO 2025-08-24 13:47:04,170 autoreload 6105 139781868523776 /workspace/file_transfer/urls.py changed, reloading.
INFO 2025-08-24 13:47:04,379 autoreload 6107 140260849012992 /workspace/file_transfer/urls.py changed, reloading.
INFO 2025-08-24 13:47:04,388 autoreload 6273 140455240851712 Watching for file changes with StatReloader
INFO 2025-08-24 13:47:04,598 autoreload 6275 139929954099456 Watching for file changes with StatReloader
INFO 2025-08-24 13:47:11,472 autoreload 6273 140455240851712 /workspace/file_transfer_system/urls.py changed, reloading.
INFO 2025-08-24 13:47:11,683 autoreload 6275 139929954099456 /workspace/file_transfer_system/urls.py changed, reloading.
INFO 2025-08-24 13:47:11,692 autoreload 6355 139724608876800 Watching for file changes with StatReloader
INFO 2025-08-24 13:47:11,896 autoreload 6357 140580739621120 Watching for file changes with StatReloader
ERROR 2025-08-24 13:49:19,891 log 6684 140359786139904 Invalid HTTP_HOST header: 'testserver'. You may need to add 'testserver' to ALLOWED_HOSTS.
Traceback (most recent call last):
  File "/workspace/venv/lib/python3.13/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/workspace/venv/lib/python3.13/site-packages/django/utils/deprecation.py", line 119, in __call__
    response = self.process_request(request)
  File "/workspace/venv/lib/python3.13/site-packages/django/middleware/common.py", line 48, in process_request
    host = request.get_host()
  File "/workspace/venv/lib/python3.13/site-packages/django/http/request.py", line 202, in get_host
    raise DisallowedHost(msg)
django.core.exceptions.DisallowedHost: Invalid HTTP_HOST header: 'testserver'. You may need to add 'testserver' to ALLOWED_HOSTS.
ERROR 2025-08-24 13:49:19,900 log 6684 140359786139904 Invalid HTTP_HOST header: 'testserver'. You may need to add 'testserver' to ALLOWED_HOSTS.
Traceback (most recent call last):
  File "/workspace/venv/lib/python3.13/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/workspace/venv/lib/python3.13/site-packages/django/utils/deprecation.py", line 119, in __call__
    response = self.process_request(request)
  File "/workspace/venv/lib/python3.13/site-packages/django/middleware/common.py", line 48, in process_request
    host = request.get_host()
  File "/workspace/venv/lib/python3.13/site-packages/django/http/request.py", line 202, in get_host
    raise DisallowedHost(msg)
django.core.exceptions.DisallowedHost: Invalid HTTP_HOST header: 'testserver'. You may need to add 'testserver' to ALLOWED_HOSTS.
ERROR 2025-08-24 13:49:19,908 log 6684 140359786139904 Invalid HTTP_HOST header: 'testserver'. You may need to add 'testserver' to ALLOWED_HOSTS.
Traceback (most recent call last):
  File "/workspace/venv/lib/python3.13/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/workspace/venv/lib/python3.13/site-packages/django/utils/deprecation.py", line 119, in __call__
    response = self.process_request(request)
  File "/workspace/venv/lib/python3.13/site-packages/django/middleware/common.py", line 48, in process_request
    host = request.get_host()
  File "/workspace/venv/lib/python3.13/site-packages/django/http/request.py", line 202, in get_host
    raise DisallowedHost(msg)
django.core.exceptions.DisallowedHost: Invalid HTTP_HOST header: 'testserver'. You may need to add 'testserver' to ALLOWED_HOSTS.
ERROR 2025-08-24 13:49:19,916 log 6684 140359786139904 Invalid HTTP_HOST header: 'testserver'. You may need to add 'testserver' to ALLOWED_HOSTS.
Traceback (most recent call last):
  File "/workspace/venv/lib/python3.13/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/workspace/venv/lib/python3.13/site-packages/django/utils/deprecation.py", line 119, in __call__
    response = self.process_request(request)
  File "/workspace/venv/lib/python3.13/site-packages/django/middleware/common.py", line 48, in process_request
    host = request.get_host()
  File "/workspace/venv/lib/python3.13/site-packages/django/http/request.py", line 202, in get_host
    raise DisallowedHost(msg)
django.core.exceptions.DisallowedHost: Invalid HTTP_HOST header: 'testserver'. You may need to add 'testserver' to ALLOWED_HOSTS.
ERROR 2025-08-24 13:49:19,926 log 6684 140359786139904 Invalid HTTP_HOST header: 'testserver'. You may need to add 'testserver' to ALLOWED_HOSTS.
Traceback (most recent call last):
  File "/workspace/venv/lib/python3.13/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/workspace/venv/lib/python3.13/site-packages/django/utils/deprecation.py", line 119, in __call__
    response = self.process_request(request)
  File "/workspace/venv/lib/python3.13/site-packages/django/middleware/common.py", line 48, in process_request
    host = request.get_host()
  File "/workspace/venv/lib/python3.13/site-packages/django/http/request.py", line 202, in get_host
    raise DisallowedHost(msg)
django.core.exceptions.DisallowedHost: Invalid HTTP_HOST header: 'testserver'. You may need to add 'testserver' to ALLOWED_HOSTS.
ERROR 2025-08-24 13:49:19,935 log 6684 140359786139904 Invalid HTTP_HOST header: 'testserver'. You may need to add 'testserver' to ALLOWED_HOSTS.
Traceback (most recent call last):
  File "/workspace/venv/lib/python3.13/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/workspace/venv/lib/python3.13/site-packages/django/utils/deprecation.py", line 119, in __call__
    response = self.process_request(request)
  File "/workspace/venv/lib/python3.13/site-packages/django/middleware/common.py", line 48, in process_request
    host = request.get_host()
  File "/workspace/venv/lib/python3.13/site-packages/django/http/request.py", line 202, in get_host
    raise DisallowedHost(msg)
django.core.exceptions.DisallowedHost: Invalid HTTP_HOST header: 'testserver'. You may need to add 'testserver' to ALLOWED_HOSTS.
ERROR 2025-08-24 13:49:19,948 log 6684 140359786139904 Invalid HTTP_HOST header: 'testserver'. You may need to add 'testserver' to ALLOWED_HOSTS.
Traceback (most recent call last):
  File "/workspace/venv/lib/python3.13/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/workspace/venv/lib/python3.13/site-packages/django/utils/deprecation.py", line 119, in __call__
    response = self.process_request(request)
  File "/workspace/venv/lib/python3.13/site-packages/django/middleware/common.py", line 48, in process_request
    host = request.get_host()
  File "/workspace/venv/lib/python3.13/site-packages/django/http/request.py", line 202, in get_host
    raise DisallowedHost(msg)
django.core.exceptions.DisallowedHost: Invalid HTTP_HOST header: 'testserver'. You may need to add 'testserver' to ALLOWED_HOSTS.
ERROR 2025-08-24 13:49:19,960 log 6684 140359786139904 Invalid HTTP_HOST header: 'testserver'. You may need to add 'testserver' to ALLOWED_HOSTS.
Traceback (most recent call last):
  File "/workspace/venv/lib/python3.13/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/workspace/venv/lib/python3.13/site-packages/django/utils/deprecation.py", line 119, in __call__
    response = self.process_request(request)
  File "/workspace/venv/lib/python3.13/site-packages/django/middleware/common.py", line 48, in process_request
    host = request.get_host()
  File "/workspace/venv/lib/python3.13/site-packages/django/http/request.py", line 202, in get_host
    raise DisallowedHost(msg)
django.core.exceptions.DisallowedHost: Invalid HTTP_HOST header: 'testserver'. You may need to add 'testserver' to ALLOWED_HOSTS.
ERROR 2025-08-24 13:49:19,967 log 6684 140359786139904 Invalid HTTP_HOST header: 'testserver'. You may need to add 'testserver' to ALLOWED_HOSTS.
Traceback (most recent call last):
  File "/workspace/venv/lib/python3.13/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/workspace/venv/lib/python3.13/site-packages/django/utils/deprecation.py", line 119, in __call__
    response = self.process_request(request)
  File "/workspace/venv/lib/python3.13/site-packages/django/middleware/common.py", line 48, in process_request
    host = request.get_host()
  File "/workspace/venv/lib/python3.13/site-packages/django/http/request.py", line 202, in get_host
    raise DisallowedHost(msg)
django.core.exceptions.DisallowedHost: Invalid HTTP_HOST header: 'testserver'. You may need to add 'testserver' to ALLOWED_HOSTS.
ERROR 2025-08-24 13:49:19,975 log 6684 140359786139904 Invalid HTTP_HOST header: 'testserver'. You may need to add 'testserver' to ALLOWED_HOSTS.
Traceback (most recent call last):
  File "/workspace/venv/lib/python3.13/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/workspace/venv/lib/python3.13/site-packages/django/utils/deprecation.py", line 119, in __call__
    response = self.process_request(request)
  File "/workspace/venv/lib/python3.13/site-packages/django/middleware/common.py", line 48, in process_request
    host = request.get_host()
  File "/workspace/venv/lib/python3.13/site-packages/django/http/request.py", line 202, in get_host
    raise DisallowedHost(msg)
django.core.exceptions.DisallowedHost: Invalid HTTP_HOST header: 'testserver'. You may need to add 'testserver' to ALLOWED_HOSTS.
ERROR 2025-08-24 13:49:19,984 log 6684 140359786139904 Invalid HTTP_HOST header: 'testserver'. You may need to add 'testserver' to ALLOWED_HOSTS.
Traceback (most recent call last):
  File "/workspace/venv/lib/python3.13/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/workspace/venv/lib/python3.13/site-packages/django/utils/deprecation.py", line 119, in __call__
    response = self.process_request(request)
  File "/workspace/venv/lib/python3.13/site-packages/django/middleware/common.py", line 48, in process_request
    host = request.get_host()
  File "/workspace/venv/lib/python3.13/site-packages/django/http/request.py", line 202, in get_host
    raise DisallowedHost(msg)
django.core.exceptions.DisallowedHost: Invalid HTTP_HOST header: 'testserver'. You may need to add 'testserver' to ALLOWED_HOSTS.
